_static/api-examples.md

```diff
@@ -126,7 +126,7 @@ You must authenticate with the CDX CROMERR service for signing a manifest. Value
 ```bash
 curl -i -X POST -H 'Content-Type: application/json' \
   --data @auth-creds.json \
-  https://e-manifest.18f.gov/api/0.1/user/authenticate
+  https://e-manifest.18f.gov/api/v1/users/authentications
```

Comment:
The Rails way assumes everything is CRUD and so the naming conventions map to that model. We have some CRUD but we also have some non-CRUD stuff (like signatures and authentication) so I don't feel like we need do make our API conform to Rails (rather than the other way around).

I.e. I'm trying to think about the API as a user of it, not as a implementer of a particular technology.